samples/HTTPParallelRequestDemo/Main.spl

```diff
@@ -0,0 +1,109 @@
+use com.ibm.streamsx.inet.http::*;
+
+/**
+ * This main composite invokes the HTTPRequestAsyncComposite cyclcal
+ * 
+ * @param	url				A list of urls url to get
+ * @param	method			The list with the method identifiers
+ * @param	data			A list with data (to push or put)
+ * @param	iterations		The number of operations to perform
+ * @param	period			The period of operations (if bult is false)
+ * @param	burst			If set to true all requests are sent in a burst
+ * @param	noConn			Number of connections
+ * 
+ */
+public composite Main {
+	param
+		expression<list<rstring>>              $url    : (list<rstring>)             getSubmissionTimeValue(
+			"url", "[\"http://httpbin.org/get\",\"http://httpbin.org/gzip\",\"http://httpbin.org/redirect/3\",\"http://httpbin.org/get\",\"http://httpbin.org/get\"]"
```

Comment:
FYI - you can use single quotes for SPL string literals, thus avoiding to have to backquote every double quote, making the sample easier to read.

Reply:
good point